snap/snapcraft.yaml

```diff
@@ -0,0 +1,55 @@
+name: anbox-cloud-gaming-demo
+base: core20
+summary: Cloud gaming demo for Anbox Cloud
+description: This demo provides an example of how to build a cloud gaming service with Anbox Cloud. It demonstrates interaction with the backend services Anbox Cloud provides and how streaming can be easily integrated.
+confinement: strict
+adopt-info: webserver
+
+apps:
+  web:
+    command: bin/start-web.sh
+    daemon: notify
+    plugs:
+      - home
+      - network
+      - network-bind
+      - process-control
+      - daemon-notify
+    environment:
+      PATH: $SNAP/bin/:$PATH
+      PYTHONPATH: $SNAP:$SNAP/lib/python3.8/site-packages
+
+parts:
+  webserver:
+    plugin: python
+    source: backend
+    python-packages:
+      - wheel
+    stage-packages:
+      - ca-certificates
+      - libsodium-dev
+    requirements:
+      - requirements.txt
+    override-build: |
+      version=unknown
+      if [ -e "$SNAPCRAFT_PART_SRC"/.snap_version ]; then
+        version=$(cat "$SNAPCRAFT_PART_SRC"/.snap_version)
+      fi
+
+      snapcraftctl set-version "$version"
```

Comment:
We have to make this work without running inside docker

Reply:
Fixed.